File: Python/1-25/07.py
import math
n = 200000
plist = [False] * 2 + [True] * n
for i in range(2, int(math.sqrt(n))+2):
    if plist[i]:
        for j in range(i**2, n+1, i): plist[j] = False
print([i for i in range(len(plist)) if plist[i]][10000])

File: C/1-25/07.c
#include <stdio.h>
#include <math.h>
int prime(int n)
{
    int i;
    for (i = 3; i < sqrt(n) + 1; i+= 2)
        if (n % i == 0)
            return 0;
    return 1;
}

int main(int argc, char *argv[])
{
    unsigned long long int i, pcount = 1;
    for (i = 3; pcount < 10001; i += 2)
        if (prime(i))
            pcount++;
   printf("%d\n", i-2);
}
